src/Servers/Reverb/Http/Response.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Laravel\Reverb\Servers\Reverb\Http;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class Response extends JsonResponse
```

Comment:
Do we want to name this specifically `ReverbResponse` so it's more clear in error logs when this response is logged?